delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/http/HttpClient.scala

```diff
@@ -0,0 +1,96 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.http
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
+import akka.http.scaladsl.unmarshalling.FromEntityUnmarshaller
+import akka.http.scaladsl.{Http, HttpExt}
+import akka.util.ByteString
+import ch.epfl.bluebrain.nexus.delta.sdk.CirceUnmarshalling._
+import ch.epfl.bluebrain.nexus.delta.sdk.http.HttpClientError._
+import io.circe.Decoder
+import monix.bio.{IO, Task}
+import monix.execution.Scheduler
+
+import scala.concurrent.TimeoutException
+import scala.reflect.ClassTag
+
+/**
+  * Http client based on the akka http model.
+  */
+trait HttpClient {
```

Comment:
you probably need to disable coverage measurement for this class since there aren't any direct tests for it.